docs/Installation.md

```diff
@@ -54,15 +54,17 @@ Omitting that will get the `master` branch which is potentially unstable.
 
 ### Install the `com.unity.ml-agents` Unity package
 
-The Unity ML-Agents C# SDK is a Unity Package. We are working on getting it added to the
-official Unity package registry which will enable you to install the `com.unity.ml-agents` package
-[directly from the registry](https://docs.unity3d.com/Manual/upm-ui-install.html) without cloning
-this repository. Until then, you will need to
-[install it from the local package](https://docs.unity3d.com/Manual/upm-ui-local.html) that you
-just cloned. You can add the `com.unity.ml-agents` package to
-your project by navigating to the menu `Window`  -> `Package Manager`. In the package manager
-window click on the `+` button. Select `Add package from disk...` and navigate into the
-`com.unity.ml-agents` folder and select the `package.json` folder.
+The Unity ML-Agents C# SDK is a Unity Package. You can install the `com.unity.ml-agents` package
+[directly from the Package Manager registry](https://docs.unity3d.com/Manual/upm-ui-install.html)
+without cloning this repository. Please make sure you enable 'Preview Packages'  in the 'Advanced'
+dropdown  in order to find it.
+
+If you want to try newer features from our development branch you
+can still [install the local package](https://docs.unity3d.com/Manual/upm-ui-local.html) from the
+repo that you just cloned. You can add the `com.unity.ml-agents` package to your project by navigating
```

Comment:
nit: maybe rearrange this to be a list? Something like

```
Add the `com.unity.ml-agents` package to your project by:
1. navigating to the menu `Window`  -> `Package Manager`.
1. In the package manager window click on the `+` button.
1. Select `Add package from disk...`
1. Navigate into the `com.unity.ml-agents` folder.
1. Select the `package.json` file.
```

Reply:
like your suggestion better. updated it